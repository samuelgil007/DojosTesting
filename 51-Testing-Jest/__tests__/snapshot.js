const cliente = {
    nombre:"samuel",
    dinero:  100,
    tipo: "premium"
};

describe("tests al ciente", () => {

    test("Que sea igual con snapshot", () => {
        expect(cliente).toMatchSnapshot();
    });

})