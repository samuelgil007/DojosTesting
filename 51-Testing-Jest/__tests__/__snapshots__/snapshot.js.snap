// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests al ciente Que sea igual con snapshot 1`] = `
Object {
  "dinero": 100,
  "nombre": "samuel",
  "tipo": "premium",
}
`;
